### PR TITLE
Add Magento CE 2.0.6

### PR DESCRIPTION
**Pull request by diazwatson**
Changes proposed in this pull request:

- Added Magento CE 2.0.6

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -152,6 +152,14 @@ commands:
...
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-2.0.6
+        package: magento/project-community-edition
+        version: 2.0.6
+        options:
+          prefer-dist: true
+          stability: alpha
+          repository-url: https://repo.magento.com
+    magento-packages:
```
Comment (tkn98) [inline config.yaml]: Duplicate line "magento-packages:", please change otherwise this should be OK!

**Comment by tkn98** [comment]
@diazwatson: Thanks for the quick PR, please see there is a little mistake with a duplicated line that breaks all other packages. Just ammend your commit and force-push, this will update the PR then.

**Comment by diazwatson** [comment]
Ups!

**Comment by tkn98** [comment]
Looks better, I think build will become green in full so merged into the development version already so it's available with `self-update --unstable` in a minute. Thanks @diazwatson!

**Comment by diazwatson** [comment]
Thank you @tkn98 for letting me know there was an error :)

**Comment by ktomk** [comment]
Magerun 1.1.11 ships with CE 2.0.6. Have fun, thanks again @diazwatson!

**Pull request merged**